atlp-rwanda/winners-c8-bn-be
# #182210060 Add search functionality

#### What does this PR do?
- Add Search functionality

#### Description of Task to be completed?
search functionality will facilitate finding a request through searching requests based on the column headers for example owner, destination, origin, start date, request status, etc.
    
#### How should this be manually tested?
- Login as a requester user
- Make sure the user has some trip requests or create some
- make a get call with the search parameters in the query to the endpoint: /trips/search?{query}
- The endpoint will return filtered trip requests according to the passed parameters

#### What are the relevant pivotal tracker stories?
- #182210060
- #182210052

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/68099724/176627402-5744e48f-9994-4d04-ac15-53109d51463f.png)

#### Questions:
